studia-historica-upsaliensa.csl

```diff
@@ -0,0 +1,588 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" and="symbol" initialize="false" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
```

Comment:
Is this style used in multiple languages  or just in English (or Swedish)? IN the latter case, we'd want a default-locale here.